app/presenters/queries/content_item_presenter.rb

```diff
@@ -0,0 +1,24 @@
+module Presenters
+  module Queries
+    class ContentItemPresenter
+      def self.present(content_item)
+        version = Version.find_by(target: content_item)
+        new(content_item, version).present
+      end
+
+      def initialize(content_item, version)
+        self.content_item = content_item
+        self.version = version
+      end
+
+      def present
+        content_item.as_json.deep_symbolize_keys
```

Comment:
I had assumed that `as_json` would provide top level symbols but you're right, it doesn't.